### ISSUE TITLE
WP Contact Widgets conflict with theme Ajax main navigation

### ISSUE THREAD

**Issue by kerschb** [godaddy/wp-contact-widgets#14]
Hello, 

I hope you can help! I am using a theme called Hazel from ThemeForest on Envato: http://themeforest.net/item/hazel-multiconcept-creative-wordpress-theme/8146099 And when I have your Contact Widget activated, it creates a problem with the main navigation, and it seems to affect only the mobile phone navigation. The drop down menus snap back up and don't stay down, so a user can't get to a sub level page. Sometime the menu works, but it is rare, or sometimes you have to click the dropdown menu 5-8 times, then it will work. Not very user-friendly. The theme writer's say their code is fine and they wanted me to check in with you and see if you can fix a problem on your end. I have a feeling you might say the same thing about your plugin, and send me back to them, but...I thought I would ask. I would love to use your plugin with my site, but if I can't get it to work with my navigation, I'll have to keep it out. 

This is my website, but currently it works okay because I have deactivated your plugin: bethkerschen.com

Thank you!
Beth

**Comment by fjarrett** [comment]
Hi @kerschb! Thanks for reporting this, I'm happy to help.

1. Can you please email me a copy of this theme ZIP so we can try to reproduce? My email address is on my GitHub profile.
2. Also can you please list all of the plugins you have activated on your site?

**Comment by kerschb** [comment]
Long story short, I did some further testing and I am starting to think it is a different plugin that is causing my problem. The problem is very intermittent, so it has been hard to test for. The bummer is, when the problem happens it can be a big problem for the user. Anyway, I think it is the Instagram Feed Pro plugin now. 

I contacted then and they just sent me an update to their plugin to fix the problem...I'll try to report back and tell you if it worked. 

Thanks again for getting back to me and offering to help,
Beth
